test/Dockerfile.jammy

```
@@ -42,6 +42,9 @@ RUN install-tool flux v0.40.2
 # renovate: datasource=docker versioning=docker
 RUN install-tool golang 1.20.1
 
+# renovate: datasource=github-releases packageName=kubernetes-sigs/kustomize
+RUN install-tool kustomize v5.0.0
```

```suggestion
RUN install-tool kustomize 5.0.0
```
you already remove the v while extract version